### ISSUE TITLE
Vespucci on Android keeps crashing on startup

### ISSUE THREAD

**Issue by olarroche** [MarcusWolschon/osmeditor4android#966]
## Vespucci Version

13.1.2.0
## Download source

Google play store
## Device (Manufacturer and Model)

Samsung Galaxy Note 5 N920C
## Android Version 

7.0
## Behaviour/Symptoms

After I used it once to successfully update an OSM POI, now when I try to use it again I only get the startup splash screen for 10 to 20 seconds (with some network traffic going on meanwhile if I have Internet connectivity up and running) and then I'm sent back to the springboard (which, btw, seems to also crash because it takes some time to start up from an empty background screen).
## Expected Behaviour

Well, ... I expect the app to start up, and to maybe display some alert if something goes wrong, instead of just crashing...
## How to recreate

Just touch the app icon in the springboard.
## Crash dump submitted (no or yes + date)

No because I don't know where to look for it on my device, if one is generated at all.

**Comment by simonpoole** [comment]
I'm not quite sure what the "springboard" is, but this sounds weird. I would suspect that something is corrupted on device (either the app image or setup data and stored state). I would suggest the following:

- remove all app-data in the Android preferences app management, start the app (it should go through the initial setup process),
- if that doesn't help, re-install

There are no known issues with Note 5s and Android 7, matter of fact it should simply run perfectly.

Network traffic: the only thing Vespucci accesses the network for  on startup (assuming you don't have auto-download turned on) are imagery tiles.

Crash dump: except if you have turned them off, you would know if you had been asked to submit one, the absence of this happening is a good indicator that the issue doesn't actually have anything to do with the app itself.

**Comment by olarroche** [comment]
• Thank you simonpoole, I deleted the Vespucci local data as you told me and it solved my problem so far. I could add my first new road chunk to OSM !
• "Springboard" is maybe macintosh parlance ? I'm talking about the default screen with the app icons.
• I don't understand how the data could be corrupted right after I started using the app ; however it might indeed be related with other problems I have with this device, e.g. that the stylus does not work anymore outside of a the small region at the bottom of the screen next to the main button.
